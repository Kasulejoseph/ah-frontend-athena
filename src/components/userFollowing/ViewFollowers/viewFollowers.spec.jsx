import React from 'react';
import { shallow } from 'enzyme';
import Followers from './viewFollowers';

const props = {
  image: '',
  username: '',
  bio: '',
}

describe('test viewing followers', ()=> {
  let wrapper;
  beforeEach(()=>{
    wrapper = shallow(<Followers {...props} />)
  })
  it('renders followers page', () =>{
    expect(wrapper).toMatchSnapshot()
  })
})