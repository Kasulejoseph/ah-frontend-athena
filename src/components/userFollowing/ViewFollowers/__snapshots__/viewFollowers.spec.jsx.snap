// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test viewing followers renders followers page 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <viewFollowers
    bio=""
    image=""
    username=""
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="img-div"
        >
          <img
            alt="none"
            className="img-cls"
            src=""
          />
        </div>,
        <div
          className="name-div"
        >
          <p
            className="u-name"
          >
            
          </p>
          <p
            className="u-bio"
          >
            
          </p>
        </div>,
      ],
      "className": "wrapper",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <img
            alt="none"
            className="img-cls"
            src=""
          />,
          "className": "img-div",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "none",
            "className": "img-cls",
            "src": "",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p
              className="u-name"
            >
              
            </p>,
            <p
              className="u-bio"
            >
              
            </p>,
          ],
          "className": "name-div",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "",
              "className": "u-name",
            },
            "ref": null,
            "rendered": "",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "",
              "className": "u-bio",
            },
            "ref": null,
            "rendered": "",
            "type": "p",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="img-div"
          >
            <img
              alt="none"
              className="img-cls"
              src=""
            />
          </div>,
          <div
            className="name-div"
          >
            <p
              className="u-name"
            >
              
            </p>
            <p
              className="u-bio"
            >
              
            </p>
          </div>,
        ],
        "className": "wrapper",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <img
              alt="none"
              className="img-cls"
              src=""
            />,
            "className": "img-div",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "none",
              "className": "img-cls",
              "src": "",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p
                className="u-name"
              >
                
              </p>,
              <p
                className="u-bio"
              >
                
              </p>,
            ],
            "className": "name-div",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "",
                "className": "u-name",
              },
              "ref": null,
              "rendered": "",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "",
                "className": "u-bio",
              },
              "ref": null,
              "rendered": "",
              "type": "p",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
